docs/Incompatibilities.md

```diff
@@ -0,0 +1,127 @@
+Incompatibilities between SGX-LKL and x86-64 Linux
+==================================================
+
+SGX-LKL is a port of Linux to a paravirtualised SGX environment, using LKL as a porting layer.
+SGX-LKL runs x86-64 programs and these programs execute the same machine instructions as they would outside of an enclave.
+The kernel; however, runs in quite a different environment to ring 0 on an x86-64 CPU and this can lead to some important differences.
```

Comment:
I noticed this in another place as well, you use a semicolon before "however". Is that correct? I would have used a comma instead.

Reply:
That's what my copyeditor tells me to use for this construct.